test/handler/test_vulture_handler.vader

```diff
@@ -0,0 +1,28 @@
+Before:
+  runtime ale_linters/python/vulture.vim
+
+After:
+  Restore
+
+  call ale#linter#Reset()
+
+  silent file something_else.py
+
+Execute(Basic vulture check should be handled):
+  call ale#test#SetFilename('something_else.py')
+  AssertEqual
+  \ [
+  \   {
+  \     'lnum': 34,
+  \     'text': "unused variable 'foo' (60% confidence)",
```

Comment:
For strings containing single quotes, just escape the single quotes by doubling them. `'foo '' bar'` -> `foo ' bar`.